prisma/schema.prisma

```diff
@@ -221,6 +222,16 @@ model PaymentActionData {
   PaymentRequestCurrent PaymentRequest?   @relation("NextAction")
 }
 
+model PaymentActionHistory {
```

Comment:
Ca you use the existing model? As we want all fields in the history too?

if you create a new model it is ok, but then include all fields